### PR TITLE
Email validation

### PR DESCRIPTION
**Pull request by AntonioGargaro**
There is now helper text and validation to ensure an email format is followed for login page

### CODE REVIEW

**Review by joaoviana — APPROVED** [review]

**Review by TKostrzewski — APPROVED** [review]

File: client/src/pages/AdminLogin/utils.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/* eslint-disable import/prefer-default-export */
+
+// eslint-disable-next-line no-control-regex
+export const emailRegex = /(?:[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*|"(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21\x23-\x5b\x5d-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])*")@(?:(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?|\[(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?|[a-z0-9-]*[a-z0-9]:(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21-\x5a\x53-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])+)\])/;
```
Comment (thomas-galligan) [inline client/src/pages/AdminLogin/utils.js]: can this go in the client utils folder?

Comment (thomas-galligan) [reply client/src/pages/AdminLogin/utils.js]: could we even restrict it to Beamery email addresses?

File: server/dummyData/dummyAdmins.js [hunk 2]
```diff
@@ -8,4 +8,11 @@ module.exports = [
     type: 'admin',
     employeeId: '123123',
   },
+  {
+    _id: ObjectID('519f1f77bcb16cd788439199'),
+    username: 'steven.bianchi@beamery.com',
+    password: 'admin',
+    type: 'admin',
+    employeeId: '1',
+  },
```
Comment (thomas-galligan) [inline server/dummyData/dummyAdmins.js:17]: dummyAdmins on the db just has the second admin object here, which I think is fine for now?

Comment (thomas-galligan) [reply server/dummyData/dummyAdmins.js:17]: in fact, it's preferable since the first login will fail your new Regex

Comment (AntonioGargaro) [reply server/dummyData/dummyAdmins.js:17]: Correct! First admin login is redundant but have kept in incase tests use it.

File: client/src/pages/AdminLogin/utils.js (hunk 1 above)
Comment (AntonioGargaro) [reply client/src/pages/AdminLogin/utils.js]: I can swap to default utils folder but not sure about beamery domain.

Comment (AntonioGargaro) [reply client/src/pages/AdminLogin/utils.js]: I don't know how to alter this regex so I'm not going to

**Pull request merged**